### DISCUSSION TITLE
Obsidian integration

### DISCUSSION THREAD

**Discussion by jmduke** [buttondown/discussions#49] [Show & tell]
Not mine, but I felt like I had to share because it's so cool: @caro401 built an Obsidian plugin for integrating with Buttondown!

https://github.com/caro401/obsidian-buttondown

**Comment by j333t** [comment]
Yea, I'm a heavy workflowy user. 
Was thinking of building an integration for that when I get some time :)

**Reply by jmduke** [reply]
<3 if there's anything we can do to make it easier, let me know!
